### PR TITLE
[#171207418] Module dns_zone first import

### PR DESCRIPTION
**Pull request by rchessa**

### CODE REVIEW

File: azurerm_dns_zone/outputs.tf [hunk 1]
```diff
@@ -0,0 +1,14 @@
+output "dns_zone_name" {
```
Comment (uolter) [inline azurerm_dns_zone/outputs.tf]:
```suggestion
output "name" {
```

Maybe just name? I don't know if there is a naming convention here, but I saw in here https://github.com/pagopa/io-infrastructure-modules-new/blob/master/azurerm_subnet/outputs.tf if was just name.

File: azurerm_dns_zone/outputs.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  description = "The full DNS name of the new zone."
+  value       = var.dns_zone_name
+}
+
+output "dns_zone_id" {
```
Comment (uolter) [inline azurerm_dns_zone/outputs.tf]:
```suggestion
output "id" {
```
may be just id ? See the comment above

File: azurerm_dns_zone/vars.tf [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+variable "region" {
+  type = string
+}
+
+variable "resource_group_name" {
+  type = string
+}
+
+variable "dns_zone_name" {
```
Comment (francescopersico) [inline azurerm_dns_zone/vars.tf]:
```suggestion
variable "name" {
```

Comment (francescopersico) [reply azurerm_dns_zone/vars.tf]: @rchessa please fix.

Comment (francescopersico) [reply azurerm_dns_zone/vars.tf]: Fixed

**Pull request merged**